app/talk/popular-tags.cjsx

```diff
@@ -0,0 +1,35 @@
+React = require 'react'
+PromiseRenderer = require '../components/promise-renderer'
+{makeHTTPRequest} = require 'json-api-client'
+{Link} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'TalkPopularTags'
+
+  propTypes:
+    header: React.PropTypes.object             # Optional header ex: <h1>Tags</h1>
+    section: React.PropTypes.string.isRequired # Talk section
+    params: React.PropTypes.object.isRequired # URL params
+
+  tagsRequest: ->
+    popularTags = talkClient.root + "/tags/popular?section=#{@props.section}&limit=10"
+    makeHTTPRequest 'GET', popularTags, null, talkClient.headers
```

Comment:
Here's what we're looking at with tags

id |   name | section     | user_id | comment_id | taggable_id | taggable_type
---|--------|-------------|---------|------------|-------------|--------------
1  | 'neat' | 'project-1' | 1       | 5          | 9           | 'Subject'
2  | 'cool' | 'project-1' | 1       | 6          | NULL        | NULL
3  | 'neat' | 'project-1' | 2       | 7          | 10          | 'Subject'
4  | 'cool' | 'project-2' | 1       | 8          | NULL        | NULL

If you find tags for `/tags?section=project-1` you'll get back the individual tags for project-1

If you have 10,000 of those tags, you're not going to iteratively count the uses or denormalize the counter to all of the records.

The way you do something like this is by aggregating the results in the database using something like `group by name`.  The result is that each row in the grouped result will represent an arbitrary number of actual rows.

If you need an array of tags with fake id's and NULL attributes, then I can respond with that, but it's just format consistency at that point.